{"name": "test_incorrect[incorrect_data2]", "status": "passed", "attachments": [{"name": "log", "source": "d07fcaa9-c7fc-4b80-a54e-d4a2cfbaf358-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "incorrect_data", "value": "{'username': '1303845892', 'password': 'nmb_python', 'check': '用户不存在'}"}], "start": 1594725660759, "stop": 1594725663608, "uuid": "4f8bc684-d27d-421c-b354-a45d9a664d4b", "historyId": "9a23aa568774cb1eb1113c9dc5db5449", "testCaseId": "486418aa0944ed0ecb75b40c2b58e369", "fullName": "TestCases.test_login.TestLogin#test_incorrect", "labels": [{"name": "tag", "value": "@pytest.mark.usefixtures('init_driver')"}, {"name": "tag", "value": "login"}, {"name": "parentSuite", "value": "TestCases"}, {"name": "suite", "value": "test_login"}, {"name": "subSuite", "value": "TestLogin"}, {"name": "host", "value": "DESKTOP-KFH4O7N"}, {"name": "thread", "value": "12400-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "TestCases.test_login"}]}